front/src/Api/Events/LeaveMucEvent.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from "zod";
+
+export const isLeaveMucEvent = z.object({
```

Comment:
@cesar-cardinale this file is used in `front/src` or already moved in `front/chat/src`?